src/palette-panel/components/PalletePanel.tsx

```diff
@@ -88,6 +89,9 @@ export const PalettePanel = () => {
         <Tab label='UI Schema' data-cy='uischema-tab' />
       </Tabs>
       <TabContent index={0} currentIndex={selectedTab}>
+        <div style={{ marginBottom: 10 }}>
```

Comment:
we should rather fix the layout of the TabContent then wrap manually

Reply:
Again please use `makeStyle`, `useStyle` or `styled` with theme support.